crates/holochain_types/src/dna/coordinator_bundle.rs

```diff
@@ -0,0 +1,68 @@
+use holochain_serialized_bytes::prelude::*;
+use holochain_zome_types::CoordinatorZomes;
+use holochain_zome_types::WasmZome;
+use holochain_zome_types::ZomeDef;
+use mr_bundle::Manifest;
+
+use crate::prelude::DnaResult;
+use crate::prelude::DnaWasm;
+
+use super::hash_bytes;
+use super::CoordinatorManifest;
+
+#[derive(
+    Debug,
+    PartialEq,
+    Eq,
+    serde::Serialize,
+    serde::Deserialize,
+    SerializedBytes,
+    shrinkwraprs::Shrinkwrap,
+    derive_more::From,
+)]
+/// A bundle of coordinator zomes.
+pub struct CoordinatorBundle(mr_bundle::Bundle<CoordinatorManifest>);
+
+impl Manifest for CoordinatorManifest {
+    fn locations(&self) -> Vec<mr_bundle::Location> {
+        self.zomes
+            .iter()
+            .map(|zome| zome.location.clone())
+            .collect()
+    }
+
+    fn path() -> std::path::PathBuf {
+        "coords.yaml".into()
```

Comment:
This seems like a misleading name, "coords" almost universally stands for "coordinates". 

Also does this have anything to do with bundling integrity and coordinator zomes together? Or is it just a bundle of coordinators?

Can we call it something boring like "zome-bundle"?

Reply:
It's just coordinators so I think `zome-bundle` would be misleading but I see the point about `coords`. Hmm, need another name for a set of coordinators. I'll have a think.